src/ansiblelint/file_utils.py

```diff
@@ -261,10 +271,15 @@ def discover_lintables(options: Namespace) -> Dict[str, Any]:
     if out is None:
         exclude_pattern = "|".join(str(x) for x in options.exclude_paths)
         _logger.info("Looking up for files, excluding %s ...", exclude_pattern)
+        # remove './' prefix from output of WcMatch
+        strip_suffix: Callable[[str], str] = lambda fname: fname[2:]
```

Comment:
What happens if for whatever reason `fname` does **not** start with `./`? Then this will chomp off something important.

I think it's better to make this a proper function and only cut the first two characters off if the string really starts with `./`.